File: MPI/mpicheck/tests/functions/2.0/MPI_Win_post.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Group var_0;
    int var_1;
    MPI_Win var_2;
    int ret;
    /* calls */
    ret = MPI_Win_post(var_0, var_1, var_2);
    ret = PMPI_Win_post(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Win_post.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_win_post(var_0, var_1, var_2, var_3)
       call pmpi_win_post(var_0, var_1, var_2, var_3)
        end program main
    